### PR TITLE
Add job id to result set footer

### PR DESCRIPTION
**Pull request by worksofliam**
Simply adds the executing job id to the footer of the result set.

image

### CODE REVIEW

**Comment by davecharron** [comment]
I wonder if the job name text should be slightly less prominent than the results status.
Perhaps font that is grey and/or slightly smaller size.
The way it is displayed in the SQL Job Manager.
image

**Comment by davecharron** [comment]
Another idea, perhaps requiring its own PR, would be to add a span to the footer, between the status and the job name, with the caption 'SQL'.  Hovering over it would display the formatted SQL that produced the results.

File: src/views/results/html.ts [hunk 1]
```diff
@@ -96,10 +97,15 @@ export function generateScroller(basicSelect: string, isCL: boolean): string {
                     appendRows(data.rows);
                   }
 
+                  if (data.jobId) {
+
+                  }
```
Comment (davecharron) [inline src/views/results/html.ts:102]: Seems there is nothing to do here, remove the block.

**Comment by worksofliam** [comment]
@davecharron 

> Edited: Alternate idea would be to have the new span have the caption 'Details', that when hovered over, shows the job name and SQL. This approach de-clutters the footer, and makes adding details later more straightforward.

I like this idea a lot, but it should be a separate PR. Also, I don't know if from a web view we get the same cool hover support like we do with tree views.

File: src/views/results/resultSetPanelProvider.ts [hunk 2]
```diff
@@ -41,9 +41,12 @@ export class ResultSetPanelProvider implements WebviewViewProvider {
 
           let queryResults = queryObject.getState() == QueryState.RUN_MORE_DATA_AVAILABLE ? await queryObject.fetchMore() : await queryObject.run();
 
+          const jobId = queryObject.getHostJob().id;
+
           data = queryResults.data;
```
Comment (davecharron) [inline src/views/results/resultSetPanelProvider.ts:47]: Doesn't look like we need `data`.  It's defined on line 29 and set here, but is never used.

**Review by davecharron — APPROVED** [review]

**Review by davecharron — APPROVED** [review]
Send it.

**Pull request merged**